cfme/provisioning.py

```diff
@@ -38,10 +38,14 @@ def select_security_group(sg):
     val = sel.get_attribute(
         "//select[@id='environment__security_groups']/option[normalize-space(.)='%s']" % sg,
         'value')
-    sel.browser().execute_script(
-        "$j('#environment__security_groups').val('%s');"
-        "$j.ajax({type: 'POST', url: '/miq_request/prov_field_changed/new',"
-        " data: {'environment__security_groups':'%s'}})" % (val, val))
+    try:
```

Comment:
Would it not be better to do this based on version then? I think that's nicer than a try except? Especially with the error you are catching it's quite a generic error.